src/ensemble/random_forest_classifier.rs

```diff
@@ -79,6 +80,8 @@ pub struct RandomForestClassifierParameters {
     pub m: Option<usize>,
     /// Whether to keep samples used for tree generation. This is required for OOB prediction.
     pub keep_samples: bool,
+    /// Seed used for bootstrap sampling and feature selection for each tree.Failed
```

Comment:
I think that we should remove `.Failed` here at the end

Reply:
Definitely! Not sure how I missed this. Thanks for the catch.